farmer/src/lib.rs

```diff
@@ -157,6 +174,15 @@ mod builder {
         /// Maximum number of pieces in single sector
         #[builder(default)]
         pub max_pieces_in_sector: Option<u16>,
+        /// Farming thread pool size
+        #[builder(default = "available_parallelism()")]
+        pub farming_thread_pool_size: usize,
+        /// Plotting thread pool size
+        #[builder(default = "available_parallelism()")]
+        pub plotting_thread_pool_size: usize,
+        /// Replotting thread pool size
+        #[builder(default = "available_parallelism()")]
+        pub replotting_thread_pool_size: usize,
```

Comment:
This is incorrect, one of the key features was to decrease number of threads for replotting to half of CPU cores available. This kind of defeats the purpose.